src/reducers/AccountReducer.js

```diff
@@ -37,7 +37,7 @@ const accountInner: Reducer<AccountState, Action> = combineReducers({
         return { ...referral, promotions }
       }
       case 'ACCOUNT_SWAP_IGNORED': {
-        return { ...state, ignoreAccountSwap: true }
+        return { ...state, ignoreAccountSwap: action.data }
```

Comment:
This does not look relevant to promotion settings. Was this a bug fix?

Reply:
No, it's a new feature. I added the ability to cancel the user preferred swap setting and go back to the affiliated one, but then I realized that was stupid & unnecessary & removed the UX but left the action.